build.gradle

```diff
@@ -40,7 +40,7 @@ repositories.mavenCentral()
 subprojects {
     apply plugin: 'biz.aQute.bnd.builder'
     apply plugin: 'java-library'
-    compileJava.options.compilerArgs += ['--release', '11']
+    compileJava.options.compilerArgs += ['--release', '20']
```

Comment:
what's the reason for it? unless there's a good reason to bump the code compatibility, I'd like to require the minimal version that's still supported so it would be easier for others to use the code also on platforms that don't have latest JRE

Reply:
It was for theme loader refactor to replace if-else with switch/case with arrows. This was introduced in java 14, I set 20 cause I thought that 20 is our target right now.

Reply:
set to 14 in PR update

Reply:
right, we bundle Java 20 in the rpm, deb, msi and dmg packages but keep the oldest Java version for both the other packages we build and for rebuilders, like [this](http://slackbuilds.org/repository/15.0/system/mucommander/). changing to Java 14 is better but it would be even better to keep Java 11 for the time being, we can still have switch-case but without the arrows..

Reply:
PR updated - converted arrow switches to old style switches (using idea), plus reverted to 11